quickwit-cli/Cargo.toml

```diff
@@ -53,4 +54,6 @@ serial_test = "0.5.1"
 
 [features]
 ci-test = []
-release-feature-set = ["quickwit-metastore/postgres", "quickwit-indexing/kafka"]
+openssl-support = ["openssl-probe"]
```

Comment:
openssl

Reply:
The name of the feature will become the name of a dependency. `openssl`

Reply:
Ok. Makes sense. This is why some projects just call the feature `ssl` then.